LICENSE

```diff
@@ -1,3 +1,4 @@
+TODO: Review license changes from 8.0.22
```

Comment:
mysql-connector-j made some modifications to their license in 8.0.22, which aren't included in this PR; does our license file need changes to match the ones made in 8.0.22?